docs/manager/graphql-reference/schema.graphql

```diff
@@ -16,10 +16,10 @@ type Queries {
   agent_summary_list(limit: Int!, offset: Int!, filter: String, order: String, scaling_group: String, status: String): AgentSummaryList
 
   """Added in 24.12.0."""
-  domain_node(id: GlobalIDField!, permission: DomainPermissionValueField = "read_attribute"): DomainNode
+  domain_node(id: GlobalIDField!, permission: DomainPermission = READ_ATTRIBUTE): DomainNode
```

Comment:
We will not be making any breaking changes to the schema.

Reply:
Should we close BA-602 and PR as "Not Planned"?